src/test/skript/tests/syntaxes/expressions/2582-Max Durability Expression.sk

```diff
@@ -0,0 +1,5 @@
+test "max durability":
```

Comment:
Should follow naming conventions described in test system README. Otherwise, looks good to me.